tailormap-components/projects/core/src/lib/feature-form/form-tree/form-tree.component.ts

```diff
@@ -72,9 +71,14 @@ export class FormTreeComponent implements OnInit, OnChanges, OnDestroy {
       },
       this.hasCheckboxes,
     );
-    if (this.features && this.features.length > 0) {
-      this.createTree(this.features);
-    }
+
+    this.store$.select(selectOpenFeatureForm).pipe(takeUntil(this.destroyed)).subscribe((features) => {
```

Comment:
I see the name selectOpenFeatureForm was already there but its a bit misleading, I thought it was meant to show if the form is open or not but its actually selecting features? Maybe consider renaming?

Reply:
Ugh, yes, I renamed, but then did some magic with merging/rebasing and lost the changes... will rename :)